### PR TITLE
Drop sidebar

### LINKED ISSUE #1409: Convert sidebar to top nav
Break sidebar into main nav items and user dropdown menu

see WIP https://github.com/linode/manager/pull/1162

### LINKED ISSUE #1430: Reboot button overlaps sidebar when scrolling horizontally
screen shot 2017-03-21 at 11 41 54 am

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #1409, closes #1430

Secondary header bar and search can come later.

screen shot 2017-03-28 at 11 27 13 am
screen shot 2017-03-28 at 11 27 24 am
screen shot 2017-03-28 at 11 27 40 am

### CODE REVIEW

**Comment by mparke** [comment]
few minor breaks:

screen shot 2017-03-28 at 11 38 15 am
screen shot 2017-03-28 at 11 38 27 am
screen shot 2017-03-28 at 11 38 31 am

**Comment by eatonphil** [comment]
@mparke alrighty. Feedback addressed. phew

File: scss/components/Notifications.scss [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
     display: none;
     $notifications-width: 350px;
 
     &--open {
         display: block;
+        position: absolute;
+        top: $nav-height;
+        height: 100%;
+        z-index: 1001;
+        left: calc(1440px + calc(calc(100% - 1440px) / 2));
```
Comment (mparke) [inline scss/components/Notifications.scss]: should the width here be based on a breakpoint var?

Comment (eatonphil) [inline scss/components/Notifications.scss]: Good point. Will fix

File: src/components/Header.js [hunk 2]
```diff
@@ -3,161 +3,129 @@ import React, { PropTypes, Component } from 'react';
 import { LinodeLogoImgSrc } from '~/assets';
```
Comment (mparke) [inline src/components/Header.js]: why combine the logic here when it was nice and modular before? the header should not manage their combined logic

Keep notifications logic in notifications, and session menu logic in session menu,

Comment (mparke) [inline src/components/Header.js]: okay, read through this again, i'm actually okay with this setup. ignore this comment

**Comment by eatonphil** [comment]
One new addition, take it or leave it. The username and alarm stood out a lot more now against the darker (not current) header nav links. So I set the username and alarm default color to the same gray as the not-selected links. They highlight the same.

screen shot 2017-03-28 at 5 05 20 pm
screen shot 2017-03-28 at 5 05 29 pm

**Comment by mparke** [comment]
getting warnings on compilation:

```
WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'md' has value 100% which isn't greater than 100%, the value of the previous key 'sm' !
Backtrace:
        node_modules/bootstrap/scss/_variables.scss:29, in mixin `-assert-ascending`
        node_modules/bootstrap/scss/_variables.scss:146

WARNING: Invalid value for $container-max-widths: This map must be in ascending order, but key 'lg' has value 100% which isn't greater than 100%, the value of the previous key 'md' !
Backtrace:
        node_modules/bootstrap/scss/_variables.scss:29, in mixin `-assert-ascending`
        node_modules/bootstrap/scss/_variables.scss:146

WARNING: Potentially invalid value for $container-max-widths: This map must be in ascending order, but key 'xl' has value 1440px whose unit makes it incomparable to 100%, the value of the previous key 'lg' !
Backtrace:
        node_modules/bootstrap/scss/_variables.scss:27, in mixin `-assert-ascending`
        node_modules/bootstrap/scss/_variables.scss:146
```

**Comment by eatonphil** [comment]
@mparke yeah that's Bootstrap trying to set an invariant that the breakpoint values always increase in size. However, we want full width for all but the biggest screens, so this doesn't really help us out. We can safely ignore it.

**Comment by mparke** [comment]
Few minor css things:

Alignment of the dropdowns appears slightly off:

screen shot 2017-03-29 at 2 56 43 pm

Shouldn't these whole list items be clickable here (vs. just the link)?

screen shot 2017-03-29 at 2 57 00 pm

Padding appears slightly off:

screen shot 2017-03-29 at 2 57 13 pm

Alignment of text here is like 1-2px off:

screen shot 2017-03-29 at 2 59 02 pm

**Comment by eatonphil** [comment]
Courtesy of @mparke https://www.youtube.com/watch?v=2yQFvZsN43A

**Comment by eatonphil** [comment]
thank you!

**Pull request merged**